client/src/index.jsx

```diff
@@ -6,6 +6,9 @@ import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import localizedFormat from 'dayjs/plugin/localizedFormat'; // import this for localized formatting
 import 'dayjs/locale/en-gb';
```

Comment:
Can you check whether it would be possible / usefull to integrate with it ?

Reply:
Not sure what should be done here. Datetime is already formatted correctly in my language.
![grafik](https://github.com/azukaar/Cosmos-Server/assets/24832654/12c98599-e9c3-43bb-9d78-d2f96cbe7054)

For timeago.js I did implement a fix, so relative time formats work as well.